templates/displayApplicants.phtml

```diff
@@ -20,16 +21,29 @@
                 <tr>
                     <th class="col-xs-2">Name</th>
                     <th class="col-xs-4">Email</th>
-                    <th class="col-xs-2">Date of application</th>
-                    <th class="col-xs-2">Cohort date</th>
+                    <th class="col-xs-2">Date of application
+                        <div>
+                            <form class="arrowForm" method="get"><button name="sort" value="dateAsc" class="arrowBtn <?php if ($sort == 'dateAsc' OR empty($sort)) {echo 'active';}?>" type="submit"><i id="arrowDateAsc" class="fas fa-arrow-up"></i></button></form>
+                            <form class="arrowForm" method="get"><button name="sort" value="dateDesc" class="arrowBtn <?php if ($sort == 'dateDesc') {echo 'active';}?>" type="submit"><i id="arrowDateDesc" class="fas fa-arrow-down"></i></button></form>
+                        </div>
+
+                    <th class="col-xs-2">Cohort date
+                        <div>
+                            <form class="arrowForm" method="get"><button name="sort" value="cohortAsc" class="arrowBtn <?php if ($sort == 'cohortAsc') {echo 'active';}?>" type="submit"><i id="arrowCohortAsc" class="fas fa-arrow-up"></i></button></form>
+                            <form class="arrowForm" method="get"><button name="sort" value="cohortDesc" class="arrowBtn <?php if ($sort == 'cohortDesc') {echo 'active';}?>" type="submit"><i id="arrowCohortDesc" class="fas fa-arrow-down"></i></button></form>
```

Comment:
This works well, however, there is no need to have four separate (but duplicate) forms. You can use a single form element that wraps around the table to do the same thing.

Reply:
Wrapped the table in a form tag.